pulse_actions/handlers/treeherder_runnable.py

```diff
@@ -78,6 +78,10 @@ def on_runnable_job_event(data, message, dry_run, stage):
     for b in buildernames:
         LOG.info("- %s" % b)
 
+    # Handle TC tasks separately
+    tc_labels = [x for x in buildernames if x.startswith('TaskLabel==')]
```

Comment:
We're using buildernames because it is available not because it is adequately named (line 37).
I would like to look into changing Treeherder's pulse message use 'job_names' instead of 'buildernames'.
https://github.com/mozilla/treeherder/pull/1490/files#diff-3fdf58ed9a2c77e3d1f36b77a6aecfd5R107